File: server/beta-server/src/main/java/ru/fullrest/mfr/test_server/model/repository/TelegramUserRepository.java
package ru.fullrest.mfr.test_server.model.repository;

import org.springframework.data.repository.CrudRepository;
import ru.fullrest.mfr.test_server.model.entity.TelegramUser;

public interface TelegramUserRepository extends CrudRepository<TelegramUser, Integer> {
}

File: server/manager/src/main/kotlin/com/lezenford/mfr/manager/model/repository/TelegramUserRepository.kt
package com.lezenford.mfr.manager.model.repository

import com.lezenford.mfr.manager.model.entity.TelegramUser
import org.springframework.data.repository.kotlin.CoroutineCrudRepository

interface TelegramUserRepository : CoroutineCrudRepository<TelegramUser, Long> {

    suspend fun findByTelegramId(telegramId: Long): TelegramUser?
}